test/sequential/test-net-connect-local-error.js

```diff
@@ -3,28 +3,21 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
```

Comment:
Can we add a comment here explaining why `EADDRINUSE` is included? Otherwise, someone will come across the test in 8 months and remove `EADDRINUSE` and we'll be right back where we are. :-D